book/website/afterword/glossary.md

```diff
@@ -16,7 +16,8 @@ Command used to add files to the staging area. Allows the user to specify which
 
 ### Author
 
-The person/s or organization that created the project.
+Authors are considered as any active local or remote contributors to the project at the time of the project developed and finalised,
+ who had a substantial contribution to the project, until the project has been finalised or came to a stage to publish the results.
```

Comment:
```
Authors in this context are the contributors to _The Turing Way_ project who have made a substantial contribution to the project such as writing a subchapter, facilitating community interactions, maintaining project’s infrastructure, mentoring contributions, and supporting others participation.
All authors are named as co-authors on the book as a whole.
```

Reply:
Author's definition here is specifically for The Turing Way project. Please update the definition as per our acknowledgement chapter.

Reply:
I see, I was thinking it is missing some important information to cover the whole scope of authorship. The above definition you added reflects it better. I will make the change and will do the PR then. Thank you.

Reply:
Thank you so much. 💫